src/main/java/com/autonomic/tmc/auth/AuthenticationCommunicationException.java

```diff
@@ -23,6 +23,7 @@
  * Runtime exception thrown when there is a communication issue while attempting auth. IOException or
  * UnknownHostException being thrown are examples of communication issues.
  */
```

Comment:
I think adding a JavaDoc deprecated with a helpful pointer like: This functionality has been moved to x because y. "Or a use this instead as this will be removed soon"